src/models/spaces/populations/mouse-model/hawks-mice-interactive.ts

```diff
@@ -275,14 +275,32 @@ export function createInteractive(model: MousePopulationsModelType) {
       }
     }
 
+    // Insurance: Ensure there are always at least two rabbits of the correct color.
+    // We need to catch it before there are zero. Once there are zero, we can't create any out
+    // of thin air. (E.g. user eliminated one population then changed environment)
+    if (environmentColor === "white" && numWhite > 1 && numWhite < 3) {
```

Comment:
Is there a reason to prefer this to `numWhite === 2`?

Reply:
Good point! It was a window that I had been changing, and hadn't even really twigged that it had become === 2...